daemon/src/engine/framework/ConsoleHistory.h

```diff
@@ -31,18 +31,40 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef FRAMEWORK_CONSOLE_HISTORY_H_
 #define FRAMEWORK_CONSOLE_HISTORY_H_
 
+#include <string>
+#include <vector>
+#include <mutex>
```

Comment:
As I think you've noticed, Common.h is included in every file using a compiler flag (this is need for PCH)